Installers/default.build

```diff
@@ -143,19 +143,20 @@ fi
     <!-- Linux Installer -->
     <if test="${(os == 'Unix' or os == 'MacOS') and directory::exists('../Tools/Pipeline/bin/Linux/AnyCPU/Release')}">
       
+      <!-- .deb installer -->
       <delete dir="Linux/tmp_deb" failonerror="false"/>
       <mkdir dir="Linux/tmp_deb"/>
       <copy todir="Linux/tmp_deb/DEBIAN/">
         <fileset basedir="Linux/DEBIAN"/>
       </copy>
-      <copy todir="Linux/tmp_deb/opt/monogame-pipeline">
+      <copy todir="Linux/tmp_deb/usr/lib/mono/xbuild/MonoGame/v3.0/Tools">
```

Comment:
not sure if we want to follow the same pattern as we do for the mac side of things here. We have a 
`/Library/Frameworks/MonoGame.framework` folder which has a versioned folder e.g v3.5.0.1234 and then a bunch of symlinks e.g.

v3.5.0.1234
v3.0 -> v3.5.0.1234
Current -> v3.5.0.1234

That way its possible to have multiple versions installed and switch between them by modifying the v3.0 symlink. 
Just a thought :)